README.md

```diff
@@ -1,5 +1,7 @@
 ## Building
 
+> If you don't need to build picotool yourself, you can find pre-built binaries for Windows, macOS, and Linux in the [pico-sdk-tools](https://github.com/raspberrypi/pico-sdk-tools/releases) repository
```

Comment:
"If you don't need..." sounds a bit strange - why would anybody **need** to build picotool themselves? :wink: 

Perhaps this could be shortened to something like "Pre-built binaries of picotool for Windows, macOS, and Linux can be downloaded from the [pico-sdk-tools](https://github.com/raspberrypi/pico-sdk-tools/releases) repository. But if you want to build it yourself from source..."

Hmm, it's obviously a slightly larger change, but given that we _are_ now offering pre-built binaries, perhaps the "Building" section should come _after_ the "Overview" section? Or perhaps be moved to an entirely separate `BUILDING.md` file?

Reply:
I think that keeping the Building and Usage by the SDK sections at the top is useful, as the Overview is very long and therefore people might miss the build instructions, which would probably create more issues.

> "If you don't need..." sounds a bit strange - why would anybody **need** to build picotool themselves? 😉

You might not trust binaries you've downloaded from the internet, or you might want to install picotool system-wide - both of these would require building it yourself, although maybe "If you don't want..." would be better phrasing? I think it's still the main use case for people who arrive at this repository, and building it from source should work fine across all platforms - it works fine on Windows for me

Reply:
Fair enough. But perhaps it's worth having a brief "Introduction" or "Description" at the top of `README.md`, rather than diving straight into "Building" on the very first line? :shrug: